cmd/ocsp-updater/main.go

```diff
@@ -478,10 +484,12 @@ func (updater *OCSPUpdater) missingReceiptsTick(batchSize int) error {
 			updater.log.AuditErr(fmt.Errorf("Failed to get certificate: %s", err))
 			continue
 		}
-		err = updater.pubc.SubmitToCT(cert.DER)
-		if err != nil {
-			updater.log.AuditErr(fmt.Errorf("Failed to submit certificate to CT log: %s", err))
-			continue
+		if updater.GRPCPub != nil {
+			ctx, cancel := context.WithTimeout(context.Background(), updater.GRPCTimeout)
+			_, _ = updater.GRPCPub.SubmitToCT(ctx, &pubPB.Request{Der: cert.DER})
+			cancel() // since we are in a for loop do this now instead of collecting a bunch
```

Comment:
Wait, why the cancel? The context is "dead", it's not going to be used anymore, right?